packages/virtual-core/src/index.ts

```diff
@@ -456,6 +456,7 @@ export class Virtualizer<
         this.isScrolling,
         this.range ? this.range.startIndex : null,
         this.range ? this.range.endIndex : null,
+        this.scrollRect,
```

Comment:
hmm we are reading the scrollRect in calculateRange via getSize, so if the range should be correct